src/client/Client.cc

```diff
@@ -10395,6 +10395,28 @@ void Client::C_Readahead::finish(int r) {
   }
 }
 
```

Comment:
Very minor nit, but if you could shorten the headline by one word to get it under 50 characters, it'd work better with a lot of tooling.

Reply:
Headline? I'm confused.

Reply:
I just meant the first line of the commit. (The title, etc.)